### ISSUE TITLE
CI: Run "autobuild/transifex-push-template.sh" every time when "translations" branch updated

### ISSUE THREAD

**Issue by morevnaproject** [synfig/synfig#1150]
We need to run "autobuild/transifex-push-template.sh" every time when "translations" branch updated.

**Comment by AnishGG** [comment]
Does this require to add a new job to the "translations" branch which will be just a simple call to `autobuild/transfex-push-template.sh`?

**Comment by ice0** [comment]
@AnishGG Yes, I think that additional job will be better because it gives a clearer understanding of what will happen when it is launched.

**Comment by AnishGG** [comment]
Then I can take this issue!

**Comment by morevnaproject** [comment]
Hi everyone! I am afraid that the job at its current implementation will fail. Did anyone checked if it works? ^__^

**Comment by AnishGG** [comment]
When I opened a PR to `synfig:translations` branch, the job was failing. But @ice0 suggested that it will fail because of an error in .po file.
Here is the job from that PR: https://travis-ci.com/github/synfig/synfig/jobs/296861674

**Comment by ice0** [comment]
@morevnaproject sure, i tested it. Job works fine.

**Comment by ice0** [comment]
Ok. We need to add one more step and install transifex binary. The whole process is described here:
https://docs.transifex.com/transifex-github-integrations/github-tx-client

@morevnaproject can you do a test project or something where we can push translations for testing purposes?

Also I suggest to add one more job to do translations validation.

**Comment by morevnaproject** [comment]
> Ok. We need to add one more step and install transifex binary. The whole process is described here:
>
> https://docs.transifex.com/client/installing-the-client
> https://docs.transifex.com/transifex-github-integrations/github-tx-client

Oh, I found that Transifex now capable to work with GitHub directly, without CI -
https://docs.transifex.com/transifex-github-integrations/github-tx-ui

This seems to be easier than configuring tx-client. ^__^"

> Also I suggest to add one more job to do translations validation (#1250)

That's great idea! @AnishGG can you do that? ^__^

**Comment by AnishGG** [comment]
> > Also I suggest to add one more job to do translations validation (#1250)
> 
> That's great idea! @AnishGG can you do that? ^__^

Yup, I will make a P.R. regarding this soon.
